priv/migrations/add_railway_ipc_consumed_messages.rb

```diff
@@ -2,10 +2,10 @@ class AddRailwayIpcConsumedMessages < ActiveRecord::Migration
   def change
     create_table :railway_ipc_consumed_messages, id: false do | t |
       t.uuid :uuid, null: false
-      t.string :message_type, null: false
+      t.string :message_type
```

Comment:
Opted to just change this file directory instead of an additional migration file, since 1. nobody is using this code and well that is mostly the only reason. If anyone feels super strongly about needing another migration file to role this back I'm all 👂 and understand. This just seemed like the easier win and nobody has access to the code since this file was originally created.